pkg/agent/variables.go

```diff
@@ -82,7 +82,19 @@ func getCSECommandVariables(config *datamodel.NodeBootstrappingConfiguration) pa
 		"auditdEnabled":                   strconv.FormatBool(to.Bool(profile.AuditDEnabled)),
 		"configGPUDriverIfNeeded":         config.ConfigGPUDriverIfNeeded,
 		"enableGPUDevicePluginIfNeeded":   config.EnableGPUDevicePluginIfNeeded,
+		"enableTelemetry":                 cs.Properties.FeatureFlags.IsFeatureEnabled("EnableTelemetry"),
```

Comment:
Does AKS support this feature?

Reply:
We don't, this is to keep the implementation of aks-engine and have less deletion and change.

Reply:
We may enable this feature next year.

Reply:
please introduce the property in NodeBootstrappingConfiguration, instead of keep reading from cs.properties.

We should have done the cleanup earlier but let's at least not make it worse.